layout/azavea-test/index.jsx

```diff
@@ -32,12 +33,11 @@ export default function AzaveaTestLayout({ children }) {
           components.
         </p>
         <p>If there is other content nested inside this layout, it will show up here:</p>
-        {children}
+        <div className="c-widget-block">
```

Comment:
The existing style rule that causes the Renderer to appear is `.c-widget-block .widget-container`, so this nesting is necessary and without it the `AzaveaSampleViz` will stop showing up. I didn't try to develop a minimum set of rules necessary for the `Renderer` to work, but we'll most likely want to do that when we develop our layout so that we're not creating unintentional coupling between this page and the dashboard pages.